packages/utils-telemetry/package.json

```diff
@@ -65,5 +65,5 @@
     "watch": "npm run build && npm-run-all --parallel -c watch:*",
     "watch:ts": "npm run build:ts -- --watch"
   },
-  "version": "1.0.3"
+  "version": "1.1.0"
```

Comment:
You don't need to manually bump it. Packages in this repo get automatically bumped based on the commit history during the release 😊

Reply:
Ah, okay. I'll reset.